content/docs/deployment/clients/_index.md

```diff
@@ -1,328 +1,684 @@
+need to tell it where to find the Linux binary.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If the output file name is not specified (using the `--output` flag) then it
+will be auto-generated and include the version number and architecture.
+
+2. **Install the package.**
 
-Create (Most OS variants with systemctl)
 ```shell
-# velociraptor-vx.x.x-linux-amd64 --config client.config.yaml rpm client
+sudo dpkg -i velociraptor_client_amd64.deb
 ```
 
-Very old Linux systems typically do not use the modern `systemd` and
-instead use the `SYSV` initialization method. You will need to build a
-slightly different RPM for these (with different initialization scripts).
+After installation you can check the service status with the command:
+
+- `systemctl status velociraptor_client` - on systems using systemd
+- `service velociraptor_client status` - on systems using SysVinit
+
 
-Create (OS variants without systemctl and old GLIBC i.e. CentOS6)
+#### Red Hat Package
+
+1. **Create a RPM installation package with an embedded client configuration file.**
+
+{{< tabs >}} {{% tab name="Linux" %}}
 ```shell
-# velociraptor-vx.x.x-linux-amd64-musl --config client.config.yaml rpm client --use_sysv
+./velociraptor rpm client --config client.config.yaml
 ```
+{{% /tab %}}
+{{% tab name="Windows" %}}
+```shell
+velociraptor.exe rpm client --config client.config.yaml --binary velociraptor-linux-musl
+```
+
+Since the Windows binary is being used here to do the packaging work, we also
+need to tell it where to find the Linux binary.
 
-Install the package
+{{% /tab %}}
+{{% tab name="macOS" %}}
 ```shell
-# sudo rpm -i velociraptor_x.x.x_client.rpm
+./velociraptor rpm client --config client.config.yaml --binary velociraptor-linux-musl
 ```
 
-Remove the package
+Since the macOS binary is being used here to do the packaging work, we also
+need to tell it where to find the Linux binary.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If the output file name is not specified (using the `--output` flag) then it
+will be auto-generated and include the version number and architecture.
+
+
+2. **Install the package.**
+
 ```shell
-# rpm -qa | grep -i velociraptor
-# sudo rpm -e velociraptor-x.x.x.X86_64
+sudo rpm -i velociraptor_client_amd64.rpm
 ```
 
+After installation you can check the service status with the command:
+
+- `systemctl status velociraptor_client` - on systems using systemd
+- `service velociraptor_client status` - on systems using SysVinit
+
+
+### Other platforms and architectures
+
+In our official releases we also provide binaries for FreeBSD.
+
+Velociraptor is written in Golang, so it is technically possible for you to
+build it on any platform supported by Go. However Go has officially ended
+support for Windows 7 with the Go 1.20 release. This means that Windows XP,
+Windows server 2003, and Windows 7/Vista can no longer be built using a
+supported version of Go. We may make occasionally - depending on demand for it -
```

Comment:
drop one of "make"

Reply:
Done

Reply:
Has Go since dropped Windows 8 / 2012? https://github.com/golang/go/issues/57004